src/firefox/remote.js

```diff
@@ -106,7 +109,7 @@ export class RemoteFirefox {
           {to: response.addonsActor, type: 'installTemporaryAddon', addonPath},
           (response) => {
             if (response.error) {
-              return reject(new WebExtError(
+              return reject(new UsageError(
```

Comment:
This should be a `WebExtError` because the developer can't recover from it. It indicates a serious problem internal to Firefox's remote debugger.